Automattic/jetpack
# Boost: Refactor environment change hook

Fixes #23211

#### Changes proposed in this Pull Request:

* Rename hook `handle_theme_change` to `handle_environment_change`. This hook is internally used by Jetpack Boost, mainly to detect when to clear existing critical CSS.

#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* With cloud CSS turned off, change the theme. See if the regenerate critical CSS admin notice appears
* With cloud CSS on, change the theme, notice if critical CSS regenerates for the new theme.
